Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -318,10 +310,10 @@ -(void)updateDisplayOrientedPose:(ARCamera*)camera {
         displayOrientationQuat = simd_quaternion(0.0f, 0.0f, 0.0f, 1.0f);
     }
```

Comment:
Did you double check the landscape orientation modes as well?  I would think these would also need to be flipped.

Reply:
You're right, good catch.